### DISCUSSION TITLE
Fetching static json file from server takes more than 10 seconds

### DISCUSSION THREAD

**Discussion by sourceful-karlson** [vercel/next.js#40253] [Help]
We have an issue here when the site loads and fetch the static json files. Sometimes, for some requests, it takes up to 10 seconds to load. Was wondering what would be the best to diagnosis this type of issue in NextJS?

**Comment by icyJoseph** [comment]
Are these files generated by Next.js? or are they JSON files you've got in the public folder?

I guess one could say that the size of the files, or their contents can make them harder to send over the network, but that's really just a guess.
